test/api/profile.test.js

```diff
@@ -165,7 +165,7 @@ describe('DELETE /profile', () => {
       );
 
     expect(process.stdout.write).toHaveBeenCalledWith(
-      expect.stringMatching(/WARN.*delete.jwt.profileTokenMismatch/)
+      expect.stringMatching(/delete.*WARN.*jwt_profileTokenMismatch/)
```

Comment:
This is a bit weird, and this is due to how the "pretty" formatter works currently. Digging into this more made me do https://github.com/mozilla/mozlog/pull/28 today, to make it more reasonable.